reports/src/com/untangle/app/reports/EventWriterImpl.java

```diff
@@ -558,6 +558,7 @@ protected void start( ReportsApp app )
     {
         this.app = app;
         UvmContextFactory.context().newThread(this).start();
+        Thread.currentThread().setPriority(4);
```

Comment:
Comment to show context. What is max. What is higher vs. lower so we understand what it's doing